modules/custom-prometheus-metrics.adoc

```diff
@@ -0,0 +1,101 @@
+//module included in the following assemblies:
+// * configuration/monitor-acs.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="custom-prometheus-metrics_{context}"]
+= Using custom Prometheus metrics
+
+[role="_abstract"]
+You can enable some {product-title-short} product metrics and use them with a dedicated Prometheus server in another application to display custom metrics for {product-title-short}. 
+
+You can use a tool such as link:https://perses.dev/perses/docs/overview/[Perses] to consume these metrics. For example, you can create a Perses dashboard to use in the {ocp} console that displays {product-title-short} data. 
```

Comment:
I mentioned Perses in the previous paragraph, so not sure how does it play with this one.

```suggestion
You can use a tool such as link:https://perses.dev/perses/docs/overview/[Perses] to display charts based on these metrics. For example, you can create a Perses dashboard to use in the {ocp} console that displays {product-title-short} data. 
```